apps/server/src/modules/user-import/service/strategy/schulconnex-fetch-import-users.service.ts

```diff
@@ -0,0 +1,46 @@
+import { SanisResponse, SchulconnexRestClient } from '@infra/schulconnex-client';
+import { UserService } from '@modules/user';
+import { Injectable } from '@nestjs/common';
+import { UserDO } from '@shared/domain/domainobject';
+import { ImportUser, SchoolEntity, SystemEntity } from '@shared/domain/entity';
+import { EntityId } from '@shared/domain/types';
+import { UserImportSchoolExternalIdMissingLoggableException } from '../../loggable';
+import { SchulconnexImportUserMapper } from '../../mapper';
+
+@Injectable()
+export class SchulconnexFetchImportUsersService {
+	constructor(
+		private readonly schulconnexRestClient: SchulconnexRestClient,
+		private readonly userService: UserService
+	) {}
+
+	public async getData(school: SchoolEntity, system: SystemEntity): Promise<ImportUser[]> {
```

Comment:
As said before: use DOs instead of Entities for School and System. ImportUser is fine, since no DO exists yet and it would be Out of Scope to create it